cmd/loop/liquidity.go

```diff
@@ -174,6 +175,109 @@ func setRule(ctx *cli.Context) error {
 	return err
 }
 
+var setParamsCommand = cli.Command{
+	Name:        "setparams",
+	Usage:       "update the parameters set for the liquidity manager",
+	Description: "Updates the parameters set for the liquidity manager.",
+	Flags: []cli.Flag{
+		cli.Float64Flag{
+			Name: "maxswapfee",
+			Usage: "the maximum percentage of swap amount " +
+				"that we limit server swap fees to.",
+		},
+		cli.Uint64Flag{
+			Name: "maxprepay",
+			Usage: "the maximum no-show (prepay) that swap " +
+				"suggestions should be limited to.",
+		},
+		cli.Uint64Flag{
+			Name: "maxminer",
+			Usage: "the maximum miner fee that swap " +
+				"suggestions should be limited to.",
+		},
+		cli.IntFlag{
+			Name: "sweepconf",
+			Usage: "the number of blocks from htlc height that " +
+				"swap suggestion sweeps should target, used " +
+				"to estimate max miner fee.",
+		},
+		cli.Uint64Flag{
+			Name: "failurebackoff",
+			Usage: "the amount of time, in seconds, that " +
+				"should pass before a channel that " +
+				"previously had a failed swap will be " +
+				"included in suggestions.",
+		},
+	},
+	Action: setParams,
+}
+
+func setParams(ctx *cli.Context) error {
+	client, cleanup, err := getClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+
+	// We need to set the full set of current parameters every time we call
+	// SetParameters. To allow users to set only individual fields on the
+	// cli, we lookup our current params, then update individual values.
+	params, err := client.GetLiquidityParams(
+		context.Background(), &looprpc.GetLiquidityParamsRequest{},
+	)
+	if err != nil {
+		return err
+	}
+
+	var flagSet bool
+
+	if ctx.IsSet("maxswapfee") {
+		feeRate := ctx.Float64("maxswapfee")
+
+		if feeRate <= 0 || feeRate >= 1 {
+			return fmt.Errorf("fee rate must be in (0;1)")
+		}
+
+		params.MaxMinerFeeSat = uint64(
+			feeRate * float64(liquidity.FeeBase),
```

Comment:
I don't think this is correct. Let's say I configure `--maxswapfee=0.5`, that would result in a PPM value of `500_000`, which is 50%. Also I'm not sure if we want to limit us to a maximum of 0.99% of fees that we can ever charge? Maybe that should also be a constant that we can change easily and hopefully not forget.

Reply:
Setting `--maxswapfee=0.5` -> 50% is intentional, so we limit fees to somewhere between (0-100%) of the swap volume. I've updated the comment to be a bit clearer, hopefully that helps. 

We could just do PPM here, but I thought percentage was more user-friendly. Also could do a percentage integer, but I think setting fractions of a percent here is required.